pkg/api/handlers/libpod/containers_create.go

```diff
@@ -65,6 +67,10 @@ func CreateContainer(w http.ResponseWriter, r *http.Request) {
 
 	rtSpec, spec, opts, err := generate.MakeContainer(r.Context(), runtime, &sg, false, nil)
 	if err != nil {
+		if errors.Is(err, storage.ErrImageUnknown) {
```

Comment:
I can reproduce the failure in CI.  We need the check as well above under `generate.CompleteSpec()`, that let's the tests pass.

Reply:
Thanks, fixed!